docs/sources/introduction/index.md

```diff
@@ -61,8 +61,8 @@ For more information, refer to the Markdown style guide in [Markdown guide](http
 
 You can contribute content in the following ways:
 
-- [Report a problem](https://grafana.com/docs/writers-toolkit/contribute/#report-a-problem)
-- [Suggest an edit](https://grafana.com/docs/writers-toolkit/contribute/#suggest-an-edit)
+- [{{% translate "docs_feedback_report" %}}](#{{< anchorize.inline "docs_feedback_report" >}}{{ anchorize (T (.Get 0)) }}{{< /anchorize.inline >}})
```

Comment:
- Should the first item render an email address with `mailto:`?
- The second item needs a `)`

Reply:
Whoops, opened the patch in the GitHub editor and didn't check locally :) 

```suggestion
- [{{% translate "docs_feedback_report" %}}](#{{< anchorize.inline "docs_feedback_report" >}}{{ anchorize (T (.Get 0)) }}{{< /anchorize.inline >}}))
```

Reply:
I think the goal of this list is to link to the sections about the ways to provide feedback rather than to actually have the user provide feedback. I just forgot the URL part to make it an actual link.

Reply:
Updated and now as intended in [76ca6d4](https://github.com/grafana/writers-toolkit/pull/890/commits/76ca6d4bfa554080a37f931e8eab1458a4d7de8f)